tools/cldr-code/src/main/java/org/unicode/cldr/util/EmojiConstants.java

```diff
@@ -83,6 +83,17 @@ public static String getEmojiFromSubdivisionCodes(String string) {
                     .add(MALE_SIGN)
                     .add(FEMALE_SIGN)
                     .freeze();
+    public static final UnicodeSet REM_PERSON_SKIP_SET =
+            new UnicodeSet(REM_SKIP_SET)
```

Comment:
I'm not sure what "REM" stands for; guessing "remainder" since "rem" sometimes stands for that in related code. Some comments might help to clarify what these items share in common